packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -75,6 +82,9 @@ export default defineComponent<KottiComment.PropsInternal>({
 					? [translations.value.replyToLabel, userToReply.value.name].join(' ')
 					: undefined,
 			),
+			showCommentThread: computed(
+				() => props.replies.length > 0 || Boolean(userToReply.value),
```

Comment:
don't prefer Boolean() with objects
because Boolean({}) gives true

so in general i'd go for userReply.value !== null